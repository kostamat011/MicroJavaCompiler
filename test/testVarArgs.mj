program MJProgram
{
	int sum(int ... var) int i, s; {
		i = len(var) - 1;
		s = 0;
		do{
			s = s + var[i];
			i--;
		} while(i >= 0);
		return s;
	}
	
	void main() {
		print(sum(1,2,3,4,5));
		print(eol);
		print(sum(1,2));
		print(eol);
		print(sum(10,10,10,10,10,10,10,10,10,10));
	}
}
