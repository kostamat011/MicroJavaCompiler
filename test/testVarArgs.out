   15
    3
  100
Completion took 0 ms